README.md

```diff
@@ -163,6 +166,190 @@ Then the page should be accessible checking only: document-title, label
 Then the page should be accessible according to: best-practice and checking: aria-roles, definition-list
 ```
 
+# RSpec
+
+## Configuration
+
+1. Require rspec matchers: in `spec/spec_helper.rb`, `spec/support/env.rb` or similar.
+
+    ``` ruby
+    require 'axe/rspec'
+    ```
+
+2. Configure Browser/Webdriver
```

Comment:
I misunderstood the `page` variable in the examples. I assumed it was the same configured `page` method from the cucumber examples, not a user defined variable. I'll take this out.